model_compression_toolkit/core/common/mixed_precision/resource_utilization_tools/ru_methods.py

```diff
@@ -433,7 +518,7 @@ class MpRuMetric(Enum):
     """
 
     WEIGHTS_SIZE = partial(weights_size_utilization)
-    ACTIVATION_OUTPUT_SIZE = partial(activation_output_size_utilization)
+    ACTIVATION_OUTPUT_SIZE = partial(activation_maxcut_size_utilization)
```

Comment:
why replace? why not add a new value to the enum of "ACTIVATION_CUTS_SIZE", and in the ru dict that is used for MP replace the activation target method to max cut?